griptape/config/base_structure_config.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+from abc import ABC
+from attr import define, field
+from griptape.config import StructureTaskMemoryConfig
+from griptape.drivers import BasePromptDriver
+from griptape.mixins import SerializableMixin
+from griptape.schemas.base_schema import BaseSchema
+from marshmallow import class_registry
+from marshmallow.exceptions import RegistryError
+
+
+@define
+class BaseStructureConfig(SerializableMixin, ABC):
```

Comment:
New mixin.

Reply:
as we think about start/stop, should we consider what this might mean for serializing a structure, too?